### PR TITLE
inject secrets using metadata for vault items

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

File: bin/secrets/inject.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    for node in $nodes; do
+        docker cp "$file" "$node":"$path"
+    done
+
+    rm -rf "$dir"
+}
+
+remote_execute(){
+    local script="$1"
+    local nodes="${2:-$all_nodes}"
```
Comment (elopio) [inline bin/secrets/inject.sh]: where is this all_nodes defined?

**Comment by elopio** [comment]
:+1: this is really cool. I left a comment about an undefined var.
And IMO, this is now crying for python. What do you think? should we schedule some time for translation after the release?

File: bin/secrets/inject.sh (hunk 1 above)
Comment (fgimenez) [inline bin/secrets/inject.sh]: it was missing in the metadata file, fixed

**Comment by fgimenez** [comment]
> this is now crying for python

after starting with the ephemeral slaves i would prefer `go get ...` rather than `git clone...; pip install ...`, let me know wdyt

> should we schedule some time for translation after the release?

absolutely! and for adding unit and infra functional tests

**Pull request merged**